src/featurizations/graphnodefeaturization.jl

```diff
@@ -78,7 +78,21 @@ function GraphNodeFeaturization(
     GraphNodeFeaturization(afs)
 end
 
-# TODO: function to compute total vector length?
+# pretty printing, short
+function Base.show(io::IO, fzn::GraphNodeFeaturization)
+    st = "GraphNodeFeaturization encoding $(length(fzn.features)) features"
+    print(io, st)
+end
+
+# pretty printing, long
+function Base.show(io::IO, ::MIME"text/plain", fzn::GraphNodeFeaturization)
+    st = "GraphNodeFeaturization encoding $(length(fzn.features)) features:\n"
+    for feature in fzn.features
+        st = string(st, "   ", feature, "\n")
```

Comment:
Just a nitpick, but could have a `"\t"` instead of `"   "` ? Tabs provide relatively more standard and aligned outputs compared to spaces.

Reply:
yep, on it